nginx/default.conf

```diff
@@ -6,14 +6,15 @@ server {
     client_max_body_size 1000M;
 
     location /airflow/ {
-        proxy_pass http://harvester:8082;
-
-        proxy_set_header X-Forwarded-For $remote_addr;
         proxy_set_header Host $host;
         proxy_redirect off;
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";       
+        proxy_set_header Connection "upgrade"; 
```

Comment:
Why are these changes needed?
The resolver 127.0.0.11 does not exist on the cluster (and not sure why it's required at all)
the set + proxy_pass is equivalent to the first line you removed... (line 9)

Reply:
You asked me to delete the help about this changes: 
https://github.com/avdata99/ckan-cloud-docker/commit/4941f64f89064d599bde8059cf6a798d4bd4aa1c
I did this changes because this code is failing.

Reply:
Again, this resolver does not exist on the cluster and it's unclear why you would need it just for this route and not for any other route (as this configuration is working as is in the cluster).
If for some reason it's failing this way, please elaborate on the errors you're seeing.

Reply:
Back to previous configuration [here](https://github.com/avdata99/ckan-cloud-docker/commit/ed5928cb3a121857c2b5ec729f9b1bcd3a567784).